From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Sat, 22 Jan 2022 14:49:34 +0000
Subject: policykitectomy

---
 usbview.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usbview.desktop b/usbview.desktop
index 69f2bb8..627d3ea 100644
--- a/usbview.desktop
+++ b/usbview.desktop
@@ -2,7 +2,7 @@
 Name=USBView
 GenericName=USB Device Viewer
 Comment=View USB devices attached to system
-Exec=pkexec /usr/bin/usbview
+Exec=usbview
 Icon=usbview
 Terminal=false
 Type=Application
